src/tools.js

```diff
@@ -0,0 +1,37 @@
+const formData = require('./index');
+
```

Comment:
Looks like you never exported your function to use in another file. I have attached an article from MDN with will give you a better understanding of exporting objects, functions, and classes in javascript.

[Article on MDN to help with export understanding](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)

Reply:
Heard that thank you I will definitely give this a read